pkg/provider/azure_loadbalancer_test.go

```diff
@@ -6730,6 +6708,112 @@ func TestReconcileZonesForFrontendIPConfigs(t *testing.T) {
 	}
 }
 
+func TestReconcileFrontendIPConfigs(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	testcases := []struct {
+		desc          string
+		service       v1.Service
+		existingFIPs  []network.FrontendIPConfiguration
+		existingPIPs  []network.PublicIPAddress
+		status        *v1.LoadBalancerStatus
+		expectedDirty bool
+		expectedFIPs  []network.FrontendIPConfiguration
+	}{
+		{
```

Comment:
can you add a test case for reconciling existing configs owned by the service without touching the others?

Reply:
also add some negative cases please

Reply:
UT added.